source/agg-pipeline-builder/create-a-view.txt

```diff
@@ -12,5 +12,44 @@ Create a View from Pipeline Results
    :depth: 1
    :class: singlecol
 
-TODO: 
+To quickly access the results of an aggregation pipeline without having to 
+execute an aggregation pipeline, you can create a :ref:`view <compass-views>` 
+on |compass|. Views are read-only, so they can help keep your data secure by 
+limiting user access to a predefined set of results. 
+
+About This Task 
+---------------
+
+Creating a view does not save the aggregation pipeline itself. 
+
+Procedure
+---------
+
+.. procedure:: 
+   :style: normal
+
+   .. step:: Click the Save dropdown button
+
+      In the aggregation pipeline pane, click the :guilabel:`Save` drop-down 
+      menu and select :guilabel:`Create view`.
+
+      .. figure:: /images/compass/query-save-pipeline-as.png
+         :alt: Save drop-down menu
+
+   .. step:: Enter a name for your view
+
+   .. step:: Create your view 
+
+      Click the :guilabel:`Create` button to create your view. |compass-short| 
+      creates a view from your pipeline results in the same database where the 
+      pipeline was created and displays saved views with the eye icon.
+
+      .. figure:: /images/compass/view-icon.png
+         :figwidth: 330px
+         :alt: View icon
```

Comment:
Same screenshot comment as above.

Reply:
Just unresolving this comment so you'd see my response. Feel free to resolve once done. But one idea to consider for screenshots is this inline implementation. Here's an example from Ian in a [Relational Migrator page](https://docs-mongodbcom-staging.corp.mongodb.com/docs-relational-migrator/docsworker-xlarge/DOCSP-27699/projects/configure-settings/). See the gear wheel in step two. Not at all necessary, but just another option that might work really well with the eye icon.

Reply:
Oh I wasn't familiar with this!